pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: nyxx_commands
-version: 3.0.0
+version: 3.1.0
 description: A framework for easily creating slash commands and text commands for Discord using the nyxx library.
 
 homepage: https://github.com/nyxx-discord/nyxx_commands/blob/main/README.md
 repository: https://github.com/nyxx-discord/nyxx_commands
 issue_tracker: https://github.com/nyxx-discord/nyxx_commands/issues
 
 environment:
-  sdk: '>=2.14.3 <3.0.0'
+  sdk: '>=2.15.0 <3.0.0'
```

Comment:
I would rather not change min required version if you are not depending on feature in new version. I think you could just leave 2.14 here because I dont see any features from 2.15 used there

Reply:
We need constructor tearoffs for const `*GuildChannel` converters (dart complains about not having constructor tearoffs enabled if we use `snowflakeToGuildChannel<ITextGuildChannel>`, not 100% certain why) and since Dart isn't hard to update nor is the update breaking, I chose bumping the SDK version over copy-pasting the guild channel converter code.